java/src/test/java/dojo/supermarket/ReceiptPrinter.java

```diff
@@ -0,0 +1,81 @@
+package dojo.supermarket;
+
+import dojo.supermarket.model.*;
+
+import java.util.Locale;
+
+public class ReceiptPrinter {
+
+    private final int columns;
+
+    public ReceiptPrinter() {
+        this(40);
```

Comment:
create static variable to make the number more understandable 
```suggestion
    private static final int DEAFULT_COLUMNS_NUMBER = 40;
    private final int columns;

    public ReceiptPrinter() {
        this(DEAFULT_COLUMNS_NUMBER);
```